examples/database/README.md

```diff
@@ -0,0 +1,31 @@
+# Overview
+
+Read the instruction below for your specific database
+
+## PostgreSQL
+### How to run the test
+- Prerequisites:
+    If you do not have them installed already
+    - `pip3 install psycopg3` - https://www.psycopg.org/psycopg3/docs/basic/install.html
+
+    - `pip3 install locust` - https://docs.locust.io/en/stable/installation.html
+
+- Set your environment variables for: 
+    - PGHOST
+    - PGPORT
+    - PGDATABASE
+    - PGUSER
+    - PGPASSWORD
+
+    so they can be picked up from `postgres_test.py`
+
+- Also, update the methods of the UserTasks class to run queries that will hit your database.
+
+#### Run test from browser:
```

Comment:
these things arent specific to PostgresUser, so I think you can skip these 2 sections as well.